src/Redux.js

```diff
@@ -5,7 +5,7 @@ export default class Redux {
   constructor(dispatcher, initialState) {
     if (typeof dispatcher === 'object') {
       // A shortcut notation to use the default dispatcher
-      dispatcher = createDispatcher(composeStores(dispatcher));
+      dispatcher = createDispatcher(composeStores(dispatcher), initialState);
```

Comment:
Yeah ok I guessed that. And probably an options object would be better yes.

PS: we are already breaking the API ;) Better do it now then later when people already start using it :)